### DISCUSSION TITLE
Exclude: true disables shortcuts

### DISCUSSION THREAD

**Discussion by hakubo** [johnlindquist/kit#358] [Errors]
I have a script that I want to be excluded from list but want to run it via shortcut: 
```js
// Shortcut: cmd opt up
// Exclude: true

await arg("arg")
```

Problem is it does not want to run when I hit cmd opt up

Workaround for this is to remove `//Exclude: true`. Like so:

```js
// Shortcut: cmd opt up

await arg("arg")
```

**Comment by johnlindquist** [comment]
Thanks for the heads-up, I'll dig into this today.

**Reply by johnlindquist** [reply]
This should be sorted in beta.33, thanks again.

**Comment by hakubo** [comment]
Works now, Thanks!
